java/integTest/src/test/java/glide/TestUtilities.java

```diff
@@ -23,4 +25,24 @@ public static int getValueFromInfo(String data, String value) {
     public static <T> T getFirstEntryFromMultiValue(ClusterValue<T> data) {
         return data.getMultiValue().get(data.getMultiValue().keySet().toArray(String[]::new)[0]);
     }
+
+    /**
+     * Replaces 'ping-sent' and 'pong-recv' timestamps in Redis Cluster Nodes output with
```

Comment:
would be nice to explain why these are being removed for future contributors who might not know how or why this causes the tests to be flaky.